### PR TITLE
Portuguese translations

### PR DESCRIPTION
**Pull request by lgfranco**
## Description

Portuguese and Portuguese as used in Portugal translations.
cc @xster 

## Related Issues

#13452 

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require Flutter developers to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (Please read [Handling breaking changes]). *Replace this with a link to the e-mail where you asked for input on this proposed change.*
- [x] No, this is *not* a breaking change.

### CODE REVIEW

File: packages/flutter_localizations/lib/src/l10n/cupertino_pt.arb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "timerPickerHourLabelOne": "hora",
+  "timerPickerHourLabelOther": "horas",
+  "timerPickerMinuteLabelOne": "min",
+  "timerPickerMinuteLabelOther": "min",
+  "timerPickerSecondLabelOne": "seg",
+  "timerPickerSecondLabelOther": "seg",
+  "cutButtonLabel": "Cortar",
+  "copyButtonLabel": "Copiar",
+  "pasteButtonLabel": "Colar",
+  "selectAllButtonLabel": "Selecionar Tudo",
```
Comment (xster) [inline packages/flutter_localizations/lib/src/l10n/cupertino_pt.arb]: this needs to be valid json

File: packages/flutter_localizations/lib/src/l10n/generated_cupertino_localizations.dart [hunk 2]
```diff
@@ -197,6 +197,182 @@ class CupertinoLocalizationFr extends GlobalCupertinoLocalizations {
   String get todayLabel => r'aujourd' "'" r'hui';
 }
 
+/// The translations for Portuguese (`pt`).
```
Comment (xster) [inline packages/flutter_localizations/lib/src/l10n/generated_cupertino_localizations.dart]: the analyzer will give an error if this file wasn't autogenerated (instructions on top)

**Comment by HansMuller** [comment]
Waiting for this PR to land first: https://github.com/flutter/flutter/pull/32513. It provides an initial set of Cupertino translations for all supported languages (including Portuguese).

**Comment by xster** [comment]
@HansMuller can you pull from TC at some point again? I already moved these suggestions in a few weeks ago.

**Comment by HansMuller** [comment]
@xster Yes, will do

**Comment by andersonmendesdev** [comment]
I added GlobalCupertinoLocalizations.delegate and Locale ('pt'), however I still get in my cupertinoTimerPicker: 
The following assertion was thrown building CupertinoTimerPicker (dirty, dependencies:
flutter: [_LocalizationsScope- [GlobalKey # e9e55], Directionality], state: _CupertinoTimerPickerState # b4279):
flutter: A non-null String must be provided for a Text widget.
flutter: 'package: flutter / src / widgets / text.dart': Failed assertion: line 241 pos 10: 'data! = null'

**Pull request closed without merging**